vss-accessor/src/vss_error.rs

```diff
@@ -0,0 +1,77 @@
+use std::error::Error;
+use std::fmt::{Display, Formatter};
+use prost::{DecodeError, Message};
+use prost::bytes::Bytes;
+use reqwest::StatusCode;
+use crate::vss::{ErrorCode, ErrorResponse};
+
+#[derive(Debug)]
+pub enum VssError {
+    InvalidRequestError(ErrorResponse),
+    ConflictError(ErrorResponse),
+    ServerInternalError(ErrorResponse),
+    InternalError(String),
+}
+
+impl VssError {
+    pub fn build(status: StatusCode, payload: &&Bytes) -> VssError {
```

Comment:
I was tempted to use "from" or "build" here, because i thought it fit more precisely, 
i think this is more of a conversion_constructor, what do you think ?